tests/upgrades/test_client.py

```diff
@@ -110,7 +110,7 @@ def create_activation_key_for_client_registration(ak_name, client_os, org, envir
     tools_cv = tools_cv.read()  # Published CV with new version
     # Promote CV
     cvv = entities.ContentViewVersion(id=max([cvv.id for cvv in tools_cv.version])).read()
-    cvv.promote(data={'environment_id': environment.id, 'force': False})
+    cvv.promote(data={'environment_ids': environment.id, 'force': False})
```

Comment:
You sure that parameter (`environment_ids`) takes integer and not list ?

Reply:
Also, you should cherrypick this in 6.9.z once merged !

Reply:
@jyejare I am of course happy to submit test results or just add [ ] if you think best.

I made the change based on my change here https://github.com/SatelliteQE/robottelo/pull/8001 on 17 Sep 2020
and test runs after this change.
BTW, I see I did ping @devendra104  at the time https://github.com/SatelliteQE/airgun/issues/520#issuecomment-694249859